strongbox-data-service/src/test/resources/etc/conf/janusgraph-inmemory-single-parition.properties

```diff
@@ -0,0 +1,10 @@
+storage.backend = inmemory
+## schema
+schema.default = none
+schema.constraints = false
+ids.block-size = 1000
+ids.num-partitions = 1
+ids.renew-timeout = 3600000
+ids.renew-percentage = 0.1
+ids.placement=org.janusgraph.graphdb.database.idassigner.placement.StaticPlacementStratrgy
```

Comment:
As above, the nme of this class is incorrect.

Reply:
fixed